src/convertToObject.js

```diff
@@ -13,6 +13,15 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString.split(';').reduce((obj, codeline) => {
+    const propertyAndValue = codeline.split(':');
```

Comment:
I would create two variables for property key and value.
```suggestion
    const [key, value]= cssProperty.split(':');
```

Reply:
Thanks for suggestion!